lib/screens/app/explorer/proposals/proposal_details.dart

```diff
@@ -19,17 +21,38 @@ class ProposalDetailsPage extends StatefulWidget {
 }
 
 class ProposalDetailsPageState extends State<ProposalDetailsPage> {
+  VoiceModel voice;
+
   double _vote = 0;
 
   bool _voting = false;
 
+  @override
+  void didChangeDependencies() {
+    Provider.of<HttpService>(context).getVoice().then((val) {
+      setState(() {
+        voice = val;
+      });
+    });
```

Comment:
using future builder here would make more sense

Reply:
There is problem I encountered with future builder - it would make unnecessary request each time when value changed in slider